.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
         run: |
           apt update
           apt install -y sudo
+          # We need to install curl because otherwise CI breaks.
+          # https://github.com/nblockchain/conventions/actions/runs/3487648476/jobs/5835523129
```

Comment:
@realmarv the logs get cleared by GitHub very quickly, they don't stay forever, so explain in which way CI breaks please

Reply:
How is it now? I mentioned the error messages so we can investigate the error later if needed